integration-libs/s4-service/order/components/reschedule-service-order/reschedule-service-order.component.html

```diff
@@ -0,0 +1,98 @@
+<form [formGroup]="form" (submit)="rescheduleServiceOrder()">
+  <div class="cx-reschedule-service-form-fields">
+    <div class="cx-section-header d-none d-lg-block d-xl-block">
+      <h2>{{ 'rescheduleService.headerLabel' | cxTranslate }}</h2>
+    </div>
+
+    <div class="cx-section-content">
+      <p>{{ 'rescheduleService.contentLabel' | cxTranslate }}</p>
+      <div class="row">
+        <label
+          class="col-md-6"
+          [attr.aria-label]="'rescheduleService.datePickerLabel' | cxTranslate"
+        >
+          <span class="label-content">
+            {{ 'rescheduleService.datePickerLabel' | cxTranslate }}
+          </span>
+          <cx-date-picker
+            [control]="$any(form.get('scheduleDate'))"
+            [min]="(minServiceDate$ | async) ?? undefined"
+            label="rescheduleService.datePickerLabel"
+          >
+          </cx-date-picker>
+        </label>
+
+        <label
+          class="col-md-6"
+          [attr.aria-label]="'rescheduleService.timePickerLabel' | cxTranslate"
+        >
+          <span class="label-content">
+            {{ 'rescheduleService.timePickerLabel' | cxTranslate }}
+          </span>
+          <select
+            *ngIf="scheduleTimes$ | async as scheduleTimes"
+            class="form-control"
+            formControlName="scheduleTime"
+            (change)="setScheduleTime($event)"
+          >
+            <option
+              *ngFor="let time of scheduleTimes"
+              [value]="time"
+              [selected]="form.get('scheduleTime')?.value === time"
+            >
+              {{ time }}
+            </option>
+          </select>
+        </label>
+      </div>
+    </div>
+  </div>
+
+  <div class="cx-reschedule-service-details">
+    <ng-template
```

Comment:
Do you require ng-template  ? can we use ng-container?

Reply:
`ng-template` is more suitable here since we are using an `*ngIf` directive. 
Also, tried with `ng-container`, it gives an error, and does not render the content too